include/details.md

```diff
@@ -86,7 +91,7 @@ __Documentation__
 | Persistent disk space              | {{ resources.default_persistent_disk }} | {{ resources.max_persistent_disk }} |
 
 {% if technicals %}
-## Technicals
+## Technical
```

Comment:
Is the 's' wrong, when talking about several technical facts? If yes, we have to also rename the variable...

Reply:
Both are correct, there is a differnce of tone however. Technicals is unconventional and informal while technical is more formal.